examples/ptz-ctrls/client.go

```diff
@@ -0,0 +1,174 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"sync"
+	"time"
+
+	"go.viam.com/rdk/components/camera"
+	"go.viam.com/rdk/components/generic"
+	"go.viam.com/rdk/logging"
+	"go.viam.com/rdk/robot/client"
+	"go.viam.com/utils/rpc"
+
+	"atomicgo.dev/keyboard"
+	"atomicgo.dev/keyboard/keys"
+)
+
+func main() {
+	logger := logging.NewDebugLogger("client")
+	machine, err := client.New(
+		context.Background(),
+		"<MACHINE_ADDRESS>",
+		logger,
+		client.WithDialOptions(rpc.WithEntityCredentials(
+			/* Replace "<API-KEY-ID>" (including brackets) with your machine's API key ID */
+			"<API-KEY-ID>",
+			rpc.Credentials{
+				Type: rpc.CredentialsTypeAPIKey,
+				/* Replace "<API-KEY>" (including brackets) with your machine's API key */
+				Payload: "<API-KEY>",
+			})),
+	)
+	if err != nil {
+		logger.Fatal(err)
+	}
+
+	defer machine.Close(context.Background())
+	logger.Info("Resources:")
+	logger.Info(machine.ResourceNames())
+
+	// rtsp-cam-1
+	rtspCam1, err := camera.FromRobot(machine, "rtsp-cam-1")
+	if err != nil {
+		logger.Error(err)
+		return
+	}
+	rtspCam1ReturnValue, err := rtspCam1.Properties(context.Background())
+	if err != nil {
+		logger.Error(err)
+		return
+	}
```

Comment:
[suggestion]

1. I don't think you need to query the camera
2. Let's make this take the secure credentials and the ptz resource name as CLI args. That way users can use this example without needing to change the code.

Reply:
Removed camera query.

Added .env wrapper for secrets.